### DISCUSSION TITLE
How/Where can I request help for new Arduino code for my project?

### DISCUSSION THREAD

**Discussion by aisi25** [community/community#21514] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>I don’t know how to write an Arduino code and unfortunately I don’t have time to learn in the meantime, I’d like to implement an Arduino to control blast gates and dust collection in my home workshop, all codes/tutorials I found on the web does not apply to the functions I need, some are close but they needs to be modified.</p>
<p>Where can I ask so that someone can write the code and help me to apply it to my project?</p>
<p>I appreciate your support.</p>
<p>Ahmed,</p>

**Comment by mpboom** [comment]
<p>Hi,</p>
<p>If you need code written by someone else, I’d strongly suggest hiring a professional developer.</p>
<p>If you can’t or don’t want to afford that, it is really fun to do it yourself! There are plenty of online resources (<a href="https://www.arduino.cc/en/Guide/HomePage" rel="nofollow noopener">https://www.arduino.cc/en/Guide/HomePage</a>) and if you have any specific questions about your code we are right here on this board ready to assist you.</p>

**Comment by aisi25** [comment]
<p>I’ll learn what I can and try to write the code.</p>
<p>Thank you.</p>

**Comment by mpboom** [comment]
<p>Good luck! And remember, if you have any questions just post them over here!</p>

**Comment by aisi25** [comment]
<p>OK, I’ve got the code with all the functions I needed by hiring a professional, and I would like to share the sketch so that anyone can use it, but I’m having servos heating issue that I would like to share to see if it can be fixed first, what is the proper way to do that?, by posting the sketch/issue in this topic or to post as a new repository and write in the “issue” tap?</p>
<p>Ahmed,</p>

**Comment by Linacruise1467** [comment]
<p>I was also looking at this ans and your answer is really very helpful for me. </p>

**Comment by mpboom** [comment, marked as answer]
<p>Generally, very specific issues (like heating issues) are more swiftly resolved by creating an issue at the project itself - simply because there are more people there with knowledge about the project.</p>
<p>On the other hand, there are so many helpful people here as well. You are always welcome to open a topic!</p>
<p>As an alternative, you can also do both. Just make sure to cross-share the links between the two places so people can find the information quickly if the issue gets resolved at either forum.</p>

**Comment by aisi25** [comment]
<p>I will do both.</p>
<p>I appreciate your help.</p>
<p>Ahmed,</p>

**Comment by aisi25** [comment]
<p>I posted a new repository, and I think no need to open a new topic for now, the heating issue has been fixed plus other problem, but am having another issue that I couldn’t figure out its source, I posted it as an issue in the repository.</p>
<p>Do I need to do something to invite people for help?</p>
<p>Thank you</p>
<p>Ahmed,</p>
